### PR TITLE
Corrections to last README update

### PR DESCRIPTION
**Pull request by CodingDoug**

### CODE REVIEW

File: README.md
Comment (honzasp) [inline README.md:1]: On line 92, the heading should probably say "sqld instances", not "sql instances". (Github doesn't allow me to put the comment there.)

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ npm install @libsql/client
 
 ### Importing
```
Comment (honzasp) [inline README.md:19]: We can also describe how to import the package in Deno. In a Netlify Edge function, the following import works:
```
import { createClient } from "https://esm.sh/@libsql/client@[version]/web"; // replace [version] with the client version
```

And in vanilla Deno:
```
import { createClient } from "npm:@libsql/client@[version]/web"; // replace [version] with the client version
```

The installation section may also mention that in Deno environments, including Netlify Edge functions, there is no installation step.

**Review by honzasp — APPROVED** [review]
Very nice, thank for making the README much better :) I have just two small comments. Feel free to merge the PR using "bors merge".

File: README.md
Comment (CodingDoug) [reply README.md:1]: Done

File: README.md (hunk 1 above)
Comment (CodingDoug) [reply README.md:21]: Done

**Comment by honzasp** [comment]
bors merge

**Pull request merged**